src/Objects/_PVector.js

```diff
@@ -0,0 +1,283 @@
+module.exports = function(options, undef) {
+  'use strict';
+
+  const TAU = 2 * Math.PI,
```

Comment:
`const` doesn't do anything in ES5 (it's technically even less efficient due to needing to add special locks to the symbol table) so let's stick with `var` declarations instead.